README.md

```diff
@@ -83,15 +87,38 @@ python tuning/sft_trainer.py  \
 ```
 
 ### Multiple GPUs with FSDP
+
+The recommendation is to use [huggingface accelerate](https://huggingface.co/docs/accelerate/en/index) to launch multi-gpu jobs, in particular when using FSDP:
+- `accelerate` is written on top of [`torch.distributed.run`](https://github.com/pytorch/pytorch/blob/main/torch/distributed/run.py).
+- `accelerate launch` CLI highly similar to `torchrun`, spawns multiple jobs (one for each gpu).
+- tightly integrated with [huggingface Trainer](https://github.com/huggingface/transformers/blob/main/src/transformers/trainer.py).
+
+`accelerate launch` CLI to be run with specific command line arguments, see example below. Default arguments handled by passing in a 
+`--config_file` argument; see [reference docs](https://huggingface.co/docs/accelerate/en/package_reference/cli#accelerate-launch) and [fixtures/accelerate_fsdp_defaults.yaml](./fixtures/accelerate_fsdp_defaults.yaml) for sample defaults.
+
 ```bash
-torchrun \
---nnodes=1 \
---nproc_per_node=8 \ 
---master_port=1234 \
+MODEL_PATH=llama-7b-hf # Huggingface model id or path to a checkpoint
+DATA_PATH=twitter_complaints.json # Path to the dataset
+OUTPUT_PATH=out # Path to the output folder where the checkpoints are saved
+
+# MASTER_PORT and MASTER_ADDR are essential for multi node training and 
+# not needed for multi gpu in single node
+MASTER_PORT=1234 # The port at which the process with rank 0 listens to
+MASTER_ADDR=x.x.x.x # The IP addresss of the node with rank 0
```

Comment:
```suggestion
# MASTER_PORT should be set to an unused port
MASTER_PORT=1234 
```

For now I suggest to keep things more streamlined and not discuss about multi node for now

Reply:
yes agreed multinode is not on our roadmap, so good to avoid. Good catch Fabian, thank you!